server/config/routes.js

```diff
@@ -304,9 +304,10 @@ exports.configure = function (app) {
   app.get('/inventory/:uuid/donations', inventory.getInventoryDonationsById);
 
   /* Depot Management */
-
+  app.post('/depots', depot.createDepots);
```

Comment:
To be consistent with our bhima-2.x APIs, these names should be:
```js
app.get('/depots', depots.list);
app.get('/depots/:id', depots.detail);
app.post(/'depots', depots.create);
app.put('/depots/:id', depots.update);
app.delete('/depots/:id', depots.delete);
```
Could you make these change in this PR?